aea/cli/interact.py

```diff
@@ -74,7 +74,7 @@ def _run_interaction_channel():
 
     try:
         multiplexer.connect()
-        while True:
+        while True:  # pragma: no cover
```

Comment:
this code looks like primary logic
not sure should skip coverage for this while cycle

Reply:
@solarw how do you propose to test this infinite loop with ctrl+c break?

Reply:
You can extract the logic in the loop in a helper function; then just test the function